docs/deploy.md

```diff
@@ -108,6 +108,12 @@ composer install --optimize-autoloader --no-dev
 sam package ...
 # Deploy
 sam deploy ...
+# Configure (e.g. passing secret env vars - existing env vars defined in template.yaml will be replaced!)
+aws lambda update-function-configuration
+  --function-name <function-name>
+  --environment '{"Variables":{
+    "SECRET_ENV_VAR":"'"$SECRET_ENV_VAR_VALUE_FROM_CICD"'",
+  }}' > /dev/null
```

Comment:
This is interesting, however could you move this section to https://github.com/mnapoli/bref/blob/master/docs/environment/variables.md instead? That way the deployment documentation stays as simple as possible, and whenever you wonder about environment variables all the information will be in one place.

Reply:
If it moves, perhaps it's worth adding a brief link to that page nearby?

Reply:
Unsure about moving this .. seeing it as a typical use Case for automated ci/cd pipelines (e.g. review apps with database connection etc.)

I can do that nevertheless

Reply:
Right but dealing with secret environment variables should go in the "environment variables" documentation.

The deploy guide is linked from all over the place, this is a guide that most people will read when getting started with Bref. It has to cover the most basic needs.

When users want to define environment variables (public or secret) they will need to go see the "Environment variables" documentation.

If we don't do that then there are many many things we could put here, e.g. anything related to Symfony's CI/CD, Laravel as well, PHP configuration as well, etc.